src/RelayServer.ts

```diff
@@ -487,26 +456,32 @@ export class RelayServer extends EventEmitter {
     envelopingRequest: EnvelopingTxRequest
   ): Promise<RelayEstimation> {
     log.debug(
-      `EnvelopingRequest:${JSON.stringify(envelopingRequest, undefined, 4)}`
+      `***EnvelopingRequest:${JSON.stringify(envelopingRequest, undefined, 4)}`
```

Comment:
I'd remove this change. If we need a specific format for some logs, I'd create one function that format them, instead of doing it on a single log basis.
E.g.:
```
const formatLog = (msg: string) => `***${msg}`;
```

Reply:
Done.